### DISCUSSION TITLE
Homepage update error

### DISCUSSION THREAD

**Discussion by sgastol** [tteck/Proxmox#1367] [Help Needed] [answered]
It is not possible to update the LXC Homepage. Whether it is the old installation or the new one I made to replicate this error. The posted screenshot describes the error. No matter which distribution debian or ubuntu the error appears every time. 

<img width="1074" alt="Screenshot 2023-04-26 at 17 29 41" src="https://user-images.githubusercontent.com/29121453/234642236-330149a9-2af3-43ec-b761-571cc89dca68.png">

**Comment by tteck** [comment, marked as answer]
Try creating another new Homepage LXC and test the update process again.
I switched from building a release to building from source.

**Reply by sgastol** [reply]
Well done! After a fresh installation, the update process went through without any problems. 

<img width="707" alt="Screenshot 2023-04-27 at 07 40 20" src="https://user-images.githubusercontent.com/29121453/234780517-c4a47786-400e-4709-bc31-ac9627ab3ec0.png">
